src/components/BookForm.js

```diff
@@ -35,35 +29,78 @@ const BookForm = () => {
       setAuthor('');
       setCategory('');
     } catch (error) {
-      // console.error('Error posting book:', error);
       // Handle the error (e.g., display an error message to the user)
     }
   };
 
+  const styles = {
+    form: {
+      display: 'flex',
+      justifyContent: 'space-between', // Add space between the elements
+      alignItems: 'center', // Center elements vertically
+      width: '80%', // Increased width to accommodate the horizontal layout
+      marginLeft: '10%',
+    },
+    input: {
+      flex: '1', // Expand the input to take available space
+      width: '25%', // Set fixed width to 25%
+      padding: '5px',
+      margin: '5px',
+      border: '1px solid #ccc',
+      borderRadius: '4px',
+      fontSize: '16px',
+    },
+    select: {
+      flex: '1', // Expand the select dropdown to take available space
+      width: '25%', // Set fixed width to 25%
+      padding: '5px',
+      margin: '5px',
+      border: '1px solid #ccc',
+      borderRadius: '4px',
+      fontSize: '16px',
+    },
+    button: {
+      flex: '1', // Expand the button to take available space
+      width: '25%', // Set fixed width to 25%
+      padding: '12px',
+      margin: '16px',
+    },
+  };
+
   return (
-    <form onSubmit={handleSubmit}>
+    <form onSubmit={handleSubmit} style={styles.form}>
       <input
         type="text"
         placeholder="Title"
         value={title}
         onChange={(e) => setTitle(e.target.value)}
         required
+        style={styles.input}
       />
       <input
         type="text"
         placeholder="Author"
         value={author}
         onChange={(e) => setAuthor(e.target.value)}
         required
+        style={styles.input}
       />
-      <input
-        type="text"
-        placeholder="Category"
+      <select
         value={category}
         onChange={(e) => setCategory(e.target.value)}
         required
-      />
-      <button type="submit">Add Book</button>
+        style={styles.select}
+      >
+        <option value="">Select a category</option>
+        <option value="Fiction">Fiction</option>
+        <option value="Non-Fiction">Non-Fiction</option>
+        <option value="Science Fiction">Science Fiction</option>
+        <option value="Mystery">Mystery</option>
+      </select>
+
+      <Button type="submit" variant="contained" style={styles.button}>
+        Add Book
+      </Button>
     </form>
```

Comment:
- [ ] I think it would be great if you add the line break after the book list like the given design to separate between the book list and the form so please add it 🔥✨

- [ ] Kindly add the heading like the given design for ADD NEW BOOK and make it have the same color and the same font size and weight ✨👍🏻💯

- [ ] I think it would be great if you make the inputs like the given design by giving the right placeholders 💯✅ so kindly make the width for the inputs and the color of their placholders the same to make our design match the given design 🔥🚀


- Your
![image](https://github.com/Eccb7/book-store/assets/58553711/340928e8-0afb-465a-951c-647794767b45)

- Given
![image](https://github.com/Eccb7/book-store/assets/58553711/c12b4586-7e4f-49ab-8636-5974ca01f024)